MAUI/TreeMap/getting-started.md

```diff
@@ -543,7 +547,7 @@ public class PopulationViewModel
 
 ## Enable legend
```

Comment:
Add referring link details for tool tip.

Reply:
Tooltip reference link added